spec/factories/case_cycles.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :case_cycle do
```

Comment:
@katelovescode Should the factory randomly set `status`, `copay_frequency`, and/or the dates (`notified_on` etc.) ?
(e.g. by using `sample`)

And or should we define `traits` based on the status?  (Or should we wait until we really need them in other specs?)

Reply:
@katelovescode What's the default status of a `CaseCycle`?

Reply:
Oop yes, good catch, can you fix that?

Reply:
I updated and pushed the diagram